extensions/logging-json/runtime/pom.xml

```diff
@@ -37,6 +37,12 @@
             <plugin>
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-bootstrap-maven-plugin</artifactId>
+                <configuration>
+                    <runnerParentFirstArtifacts>
+                        <!-- Needed because org.jboss.logmanager.formatters.JsonFormatter loads javax.json.Json in it's constructor -->
```

Comment:
```suggestion
                        <!-- Needed because org.jboss.logmanager.formatters.JsonFormatter loads javax.json.Json in its constructor -->
```

Reply:
Good catch. Fixes